packages/react-core/src/components/Wizard/__tests__/Wizard.test.tsx

```diff
@@ -222,3 +223,91 @@ describe('Wizard', () => {
     expect(main).toHaveAttribute('aria-labelledby', 'main-aria-labelledby');
   });
 });
+
+  test('wiz with disabled steps', () => {
+    const steps: WizardStep[] = [
+      { name: 'A', component: <p>Step 1</p>},
+      { name: 'B', component: <p>Step 2</p>, isDisabled: true },
+      { name: 'C', component: <p>Step 3</p>},
+      { name: 'E', component: <p>Step 4</p>, isDisabled: true },
+      { name: 'G', component: <p>Step 5</p> },
+
+    ];
+    
+    render(
+      <Wizard
+        navAriaLabel="nav aria-label"
+        description="This wizard step is disabled"
+        steps={steps}
+      />
+    );
+
+    expect(screen.getByRole('button',{ name: "B" })).toBeDisabled();
+    expect(screen.getByRole('button',{ name: "E" })).toBeDisabled();
+  });
+
+  test('wiz skip the step disabled when press the next/back button', () => {
+    const steps: WizardStep[] = [
+      { name: 'A', component: <p>Step 1</p>},
+      { name: 'B', component: <p>Step 2</p>, isDisabled: true },
+      { name: 'C', component: <p>Step 3</p>},
+
+    ];
+    
+    render(
+      <Wizard
+        navAriaLabel="nav aria-label"
+        description="This wizard step is disabled"
+        steps={steps}
+      />
+    );
+
+    userEvent.click(screen.getByRole('button', { name: 'Next' }));
+    expect(screen.getByRole('button',{ name: "C" })).toHaveClass('pf-m-current')
+
+    userEvent.click(screen.getByRole('button', { name: 'Back' }));
+    expect(screen.getByRole('button',{ name: "A" })).toHaveClass('pf-m-current')
+  });
+  
+  test('wiz skip the step when click on the nav item disabled', () => {
+    const steps: WizardStep[] = [
+      { name: 'A', component: <p>Step 1</p>},
+      { name: 'B', component: <p>Step 2</p>, isDisabled: false },
+      { name: 'C', component: <p>Step 3</p>}
+    ];
+    
+    render(
+      <Wizard
+        navAriaLabel="nav aria-label"
+        description="This wizard step is disabled"
+        steps={steps}
+      />
+    );
+
+    const navItemButton = screen.getByRole('button',{ name: "C" })
+    
+    userEvent.click(navItemButton);
```

Comment:
Shouldn't it be the disabled step that gets clicked here?

Reply:
oh, shrap! thanks 👍🏼